utils/check_repo.py

```diff
@@ -122,6 +122,8 @@
     "TFRagTokenForGeneration",
     "Wav2Vec2ForCTC",
     "HubertForCTC",
+    "Wav2Vec2ForSequenceClassification",
```

Comment:
Yes! We have to discuss a bit with @Narsil how to best add those models to pipelines

Reply:
```
# Update this list for models that are not in any of the auto MODEL_XXX_MAPPING. Being in this list is an exception and
# should **not** be the rule.
```

Seems like the exception has grown quite a bit :)